monkey/monkey_island/cc/next_ui/src/app/(protected)/plugins/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import Link from 'next/link';
+import { useGetAvailablePluginsQuery } from '@/redux/features/api/agentPlugins/agentPluginEndpoints';
+import React from 'react';
+
+export default function Plugins() {
+    const { data, error, isLoading, isError, isSuccess } =
+        useGetAvailablePluginsQuery();
+
+    if (isSuccess) return <div>{JSON.stringify(data)}</div>;
+    if (isLoading) return <div>loading...</div>;
+    if (isError) {
+        return <div>Error: {error.data.response.errors}</div>;
+    }
+    return (
+        <div>
```

Comment:
Consider using Box component instead of div

Reply:
This is a mock/POC page, it will get deleted in the following PR's